WebSocketTest/ConnectionHandlers/ClientConnection.cs

```diff
@@ -40,7 +40,7 @@ public void Accept(Client clientData)
 			{
 				try
 				{
-					SendHandshake(clientData, Handshake.GenerateHandshake(data));
+                    MessageSender.SendToSpecific(data, clientData);
```

Comment:
This replaces the generated handshake with the received request, HandShake.GenerateHandshake is still required (but has to be adapted to return the string it encodes by removing the utf8encoding at the end) 
```suggestion
					MessageSender.SendToSpecific(Handshake.GenerateHandshake(data), clientData);
```

Reply:
Ah yes, my bad. Working on it 👍

Reply:
Just committed changes that should fix this.

Reply:
I'll review it in a moment, VS updates are taking longer than expected...